solana-programs/anchor/audius-data/scripts/build-dev.sh

```diff
@@ -5,8 +5,8 @@ cd $PROTOCOL_DIR/solana-programs/anchor/audius-data
 echo "Building and deploying audius_data program with anchor CLI..."
 anchor build
 AUDIUS_DATA_PROGRAM_ID=$(solana-keygen pubkey target/deploy/audius_data-keypair.json)
-sed -i "s/Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS/$AUDIUS_DATA_PROGRAM_ID/g" Anchor.toml
-sed -i "s/Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS/$AUDIUS_DATA_PROGRAM_ID/g" programs/audius-data/src/lib.rs
+sed -i "s/JCSN8S5SAMjj9G1vEfwGn4gVGQ8W4YS437UdPZdjALiT/$AUDIUS_DATA_PROGRAM_ID/g" Anchor.toml
+sed -i "s/JCSN8S5SAMjj9G1vEfwGn4gVGQ8W4YS437UdPZdjALiT/$AUDIUS_DATA_PROGRAM_ID/g" programs/audius-data/src/lib.rs
```

Comment:
why do we change this?

Reply:
good catch - the hardcoded value changed for me locally and wasn't shown in the diff so i thought it had changed. i can remove this